### PR TITLE
Adding Registry and Portal to networking graphic.

### PR DESCRIPTION
**Pull request by stuclem**
Per title. Also tarted up the graphic a bit, to try to distinguish better better between what is VIC, what is vSphere, and what is Docker.

@mlh78750 can you please take a look? Thanks!

### CODE REVIEW

**Comment by stuclem** [comment]
Fixes https://github.com/vmware/vic-product/issues/16.

**Comment by mlh78750** [comment]
Sorry for the delay on the review. While the position of Harbor and Admiral on the management network is a valid setup, it's not the only way this would work. The only requirement for the registry is that the endpoint VM has a usable route to the registry over any network. Practically speaking this will be the management or the public network. For Admiral, it just needs a usable route to the endpoint VM's client network's interface. @hmahmood to check my answers.

**Comment by hmahmood** [comment]
@mlh78750 is correct. You can move the Admiral box in the image to connect to the client network. The registry can be attached to the client, public, or management networks, although the latter two is the recommended approach. Not sure how we would represent that in the picture though.

**Comment by stuclem** [comment]
Thanks @mlh78750 and @hmahmood. I have moved the VIC Appliance to the left of the image, and connected the Management Portal to the client network, leaving Registry connected to the management network. I updated the text of the topic that includes this graphic to state which networks Registry/MPortal can/must connect to. Changes are in https://github.com/vmware/vic-product/pull/189/commits/6d9278e844cb825cfef244b13e6bbbfbd21db4fe. Is this OK now?

File: docs/user_doc/vic_vsphere_admin/networks.md [hunk 1]
```diff
@@ -16,33 +16,33 @@ This topic provides an overview of the different network types that virtual cont
...
-The following sections describe each of the VCH network types, that are represented in the image by green elipses. The blue shapes represent   Docker objects, and the gray shapes represent vSphere. 
+The following sections describe each of the VCH network types.
 
 **IMPORTANT**: A VCH supports a maximum of 3 distinct network interfaces. The bridge network requires its own port group, at least two of the public, client, and management networks must share a network interface and therefore a port group. Container networks do not go through the VCH, so they are not subject to this limitation. This limitation will be removed in a future release.
 
 
 ## Management Network {#management}
 
-The network for communication between the VCH and vCenter Server and ESXi hosts. The VCH uses this network to provide the `attach` function of the Docker API.
+The network for communication between the VCH, vCenter Server, and ESXi hosts. The VCH uses this network to provide the `attach` function of the Docker API. vSphere Integrated Containers Registry must be able to access VCHs either over the management network or over the public network.
```
Comment (hmahmood) [inline docs/user_doc/vic_vsphere_admin/networks.md]: Isn't this the over way around? i.e. the VCH needs to access the VIC registry over any of the networks the VCH is connected to (management and public preferred).

**Comment by stuclem** [comment]
Thanks @hmahmood. Duly updated.

**Comment by stuclem** [comment]
Thanks @hmahmood !

**Pull request merged**